### DISCUSSION TITLE
Including additional atomic properties using MACE

### DISCUSSION THREAD

**Discussion by rfhari** [ACEsuit/mace#484] [Q&A] [answered]
Hi, I recently came across [this article](https://arxiv.org/abs/2205.06643), and it seems that Equation 8 discusses the potential extension of the ACE basis functions (like in articles [paper1](https://doi.org/10.1103/PhysRevB.102.024104) and [paper2](https://www.nature.com/articles/s41524-024-01196-8#Abs1)) to include additional atomic properties, such as magnetic moments. However, if I understand correctly, the current MACE model only considers atomic positions. Could you please confirm if my understanding is correct? If so, could you also explain why options to include these additional features are not currently available?

**Comment by ilyes319** [comment, marked as answer]
Hi, @rfhari 

In principle one can easily add such atomic properties to MACE by just adding new kind of node attributes, either scalar ones or vectorial ones. Including magnetic moments is in principle very interesting, but requires special data. If your project requires such additional quantities, we would be happy to accept a PR.
